frontend/pages/index.js

```diff
@@ -41,27 +41,27 @@ const Home = () => {
    * @param ModalValue -The current value of the state
    */
   /* eslint-disable no-unused-vars*/
-  const triggerModal = (setModal, ModalValue) => {
+  function triggerModal(setModal, ModalValue) {
```

Comment:
Why change this to a `function`?

Reply:
i honestly dont remember why i had it as a function. switching back to const to be more consistent with the rest of the code.